### PR TITLE
New post: Modern JavaScript in the Browser

### PR DESCRIPTION
**Pull request by ssaunier**
@cedricmenteau it needs to be reviewed for typos. Thanks 🙏 

image

### CODE REVIEW

File: posts/2017-08-11-modern-javascript-in-the-browser.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  alert("You clicked me!");
+  // TODO: implement a modal opening logic instead of this alert :)
+});
+```
+
+You can read more about [`addEventListener`](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener) on the MDN web docs 👌
+
+## 3 - AJAX
+
+AJAX is the cornerstone of the modern Web. Thanks to this technology, a web page can be refreshed in the background with new information. You can think about Gmail which shows up new emails in the Inbox, Facebook notifying you that new content is available in your timeline, etc.
```
Comment (Eschults) [inline posts/2017-08-11-modern-javascript-in-the-browser.md]: > You can think of Gmail's new emails showing up in the Inbox

File: posts/2017-08-11-modern-javascript-in-the-browser.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+})
+  .then(response => response.json())
+  .then((data) => {
+    console.log(data); // JSON from the API response.
+  });
+```
+
+## Conclusion
+
+You can now combien these three techniques to create dynamic web pages! You will select elements thanks to one of the three methods at hand (`getElementById`, `querySelector` or the one returning an `Array`: `querySelectorAll`). With an element at hand, you can `addEventListener` on a given event type (`click`, `focus`, `blur`, `submit`, etc.). The callback passed to this listener could contain an AJAX call using `fetch`.
```
Comment (Eschults) [inline posts/2017-08-11-modern-javascript-in-the-browser.md]: combine

Comment (Eschults) [inline posts/2017-08-11-modern-javascript-in-the-browser.md]: You will select elements thanks to one of the three given methods ~~at hand~~

**Comment by ssaunier** [comment]
@cedricmenteau Ready for merge :shipit:

**Comment by cedricmenteau** [comment]
@ssaunier 🚀

**Pull request merged**